src/OmniSharp/Models/HighlightResponse.cs

```diff
@@ -0,0 +1,10 @@
+namespace OmniSharp.Models
+{
+    public class HighlightResponse
+    {
+        public int Line { get; set; }
+        public int Start { get; set; }
+        public int End { get; set; }
+        public string Kind { get; set; }
```

Comment:
It definitely *can* be an enum. Though, the way it works today is more like CSS classes, it space-separates multiple "kinds" into a string. The original implementation actually just used css classes. Not to mention that I just `ToString()` the kind I get from Roslyn, so I don't know all of the kinds one might get.